<launch>

  <node pkg="asctec_autopilot" type="circular_route" name="circular_route" args="" output="screen"/>
	<param name="radius" type="int" value="15000"/>
	<param name="lap_nb" type="int" value="1"/>
#### int_angle in degrees##################################
	<param name="int_angle" type="int" value="30"/>




##### Nodelet manager ####################################

  <node pkg="nodelet" type="nodelet" name="pelican_manager6" args="manager"/>

##### Asctec Autopilot ###################################

  <node pkg="nodelet" type="nodelet" name="AutoPilotNodelet6" 
    args="load asctec_autopilot/AutoPilotNodelet pelican_manager6">

    <param name="port" type="string" value="/dev/ttyS3"/> <!-- LL UART_D -->
    <param name="speed" type="int" value="57600"/>
    <param name="freq" type="double" value="20.0"/>

    <param name="enable_GPS_DATA" type="bool" value="true"/>
    <param name="interval_GPS_DATA" type="int" value="1"/>
    <param name="offset_GPS_DATA" type="int" value="0"/>

    <param name="enable_GPS_DATA_ADVANCED" type="bool" value="true"/>
    <param name="interval_GPS_DATA_ADVANCED" type="int" value="1"/>
    <param name="offset_GPS_DATA_ADVANCED" type="int" value="0"/>

    <param name="enable_IMU_CALCDATA" type="bool" value="true"/>
    <param name="interval_IMU_CALCDATA" type="int" value="1"/>
    <param name="offset_IMU_CALCDATA" type="int" value="0"/>

    <param name="enable_WAYPOINT" type="bool" value="true"/>
    <param name="interval_WAYPOINT" type="int" value="1"/>
    <param name="offset_WAYPOINT" type="int" value="0"/>

    <param name="enable_CURRENT_WAY" type="bool" value="true"/>
    <param name="interval_CURRENT_WAY" type="int" value="1"/>
    <param name="offset_CURRENT_WAY" type="int" value="0"/>

    <param name="enable_LL_STATUS" type="bool" value="true"/>
    <param name="interval_LL_STATUS" type="int" value="1"/>
    <param name="offset_LL_STATUS" type="int" value="0"/>

  </node>

</launch>

